### ISSUE TITLE
Cannot import 'pywintypes'

### ISSUE THREAD

**Issue by nicoddemus** [conda-forge/pywin32-feedstock#1]
(From conda-forge/pyinstaller-feedstock#7)

I'm experiencing an import error for `pywintypes`. After creating a clean environment with respective python versions and `pywin32`:

**3.5**:

```
(pywin-test-3.5) C:\Users\Bruno>conda list
# packages in environment at c:\Miniconda3-dev\envs\pywin-test-3.5:
#
certifi                   2017.1.23                py35_0    conda-forge
pip                       9.0.1                    py35_0    conda-forge
python                    3.5.3                         0    conda-forge
pywin32                   220                      py35_0    conda-forge
setuptools                33.1.0                   py35_0    conda-forge
vs2015_runtime            14.0.25420                    0    conda-forge
wheel                     0.29.0                   py35_0    conda-forge
wincertstore              0.2                      py35_0    conda-forge

(pywin-test-3.5) C:\Users\Bruno>python
Python 3.5.3 | packaged by conda-forge | (default, Jan 23 2017, 20:04:35) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> ^Z


(pywin-test-3.5) C:\Users\Bruno>python -c "import pywintypes"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "c:\Miniconda3-dev\envs\pywin-test-3.5\lib\site-packages\win32\lib\pywintypes.py", line 124, in <module>
    __import_pywin32_system_module__("pywintypes", globals())
  File "c:\Miniconda3-dev\envs\pywin-test-3.5\lib\site-packages\win32\lib\pywintypes.py", line 98, in __import_pywin32_system_module__
    raise ImportError("No system module '%s' (%s)" % (modname, filename))
ImportError: No system module 'pywintypes' (pywintypes35.dll)
```

**3.6**:

```
(pywin-test-3.6) C:\Users\Bruno>conda list
# packages in environment at c:\Miniconda3-dev\envs\pywin-test-3.6:
#
certifi                   2017.1.23                py36_0    conda-forge
pip                       9.0.1                    py36_0    conda-forge
python                    3.6.0                         1    conda-forge
pywin32                   220                      py36_0    conda-forge
setuptools                33.1.0                   py36_0    conda-forge
vs2015_runtime            14.0.25420                    0    conda-forge
wheel                     0.29.0                   py36_0    conda-forge
wincertstore              0.2                      py36_0    conda-forge

(pywin-test-3.6) C:\Users\Bruno>python
Python 3.6.0 | packaged by conda-forge | (default, Jan 14 2017, 00:31:55) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> ^Z


(pywin-test-3.6) C:\Users\Bruno>python -c "import pywintypes"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "c:\Miniconda3-dev\envs\pywin-test-3.6\lib\site-packages\win32\lib\pywintypes.py", line 124, in <module>
    __import_pywin32_system_module__("pywintypes", globals())
  File "c:\Miniconda3-dev\envs\pywin-test-3.6\lib\site-packages\win32\lib\pywintypes.py", line 98, in __import_pywin32_system_module__
    raise ImportError("No system module '%s' (%s)" % (modname, filename))
ImportError: No system module 'pywintypes' (pywintypes36.dll)
```

Not sure how the latest build managed to test successfully given that `pywintypes` is one of the modules imported during the testing phase.

**2.7** works:

```
(pywin-test-2.7) C:\Users\Bruno>conda list
# packages in environment at c:\Miniconda3-dev\envs\pywin-test-2.7:
#
certifi                   2017.1.23                py27_0    conda-forge
pip                       9.0.1                    py27_0    conda-forge
python                    2.7.12                        1    conda-forge
pywin32                   220                      py27_0    conda-forge
setuptools                33.1.0                   py27_0    conda-forge
vs2008_runtime            9.00.30729.1                  2
wheel                     0.29.0                   py27_0    conda-forge
wincertstore              0.2                      py27_0    conda-forge

(pywin-test-2.7) C:\Users\Bruno>python
Python 2.7.12 | packaged by conda-forge | (default, Sep  8 2016, 14:46:05) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> ^Z


(pywin-test-2.7) C:\Users\Bruno>python -c "import pywintypes"
```

**Comment by nicoddemus** [comment]
Update: after some debugging, I figured out that `2.7` worked initially for me because it found a `pywintypes27.dll` in my Mercurial installation. After removing Mercurial from the PATH, it failed in the same way.

Perhaps that's one clue as to why this is passing on AppVeyor, it might be because of some other software installed there.

**Comment by vmuriart** [comment]
I dealt with a similar issue yesterday, and the following was helpful to debug. 

- [`pywintypes.py`](https://github.com/vmuriart/pywin32/blob/master/win32/Lib/pywintypes.py) controls the discovery of the `dll`. 
- Printing `found` is useful to find which `pywintypes.dll` its loading. 
- In my case yesterday it was loading it from `c:\windows\system32\` which was placed there by a different python installer. New conda enviroments imported that version of `pywintypes.dll` over their local copy.
- In the case of your failing installs, did you try running the `pywin32_postinstall.py` after the setup is complete?

**Comment by nicoddemus** [comment]
Fixed by #3
